package.json

```diff
@@ -30,25 +28,23 @@
   },
   "homepage": "https://github.com/dial-once/node-bunnymq#readme",
   "dependencies": {
-    "amqplib": "^0.4.1",
+    "amqplib": "^0.5.3",
     "deserialize-error": "0.0.3",
-    "dotenv": "^2.0.0",
-    "serialize-error": "^2.1.0",
-    "uuid": "^3.0.1"
+    "dotenv": "^7.0.0",
+    "serialize-error": "^3.0.0",
+    "uuid": "^3.3.2"
   },
   "devDependencies": {
-    "assert": "1.3.0",
-    "child-process-promise": "^2.2.0",
-    "eslint": "^5.4.0",
+    "child-process-promise": "^2.2.1",
+    "eslint": "^5.15.1",
     "eslint-config-airbnb": "^17.1.0",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-jsx-a11y": "^6.1.1",
-    "eslint-plugin-react": "^7.11.1",
-    "minami": "1.1.1",
-    "mocha": "^5.2.0",
+    "eslint-plugin-import": "^2.16.0",
+    "eslint-plugin-jsx-a11y": "^6.2.1",
+    "eslint-plugin-react": "^7.12.4",
+    "mocha": "^6.0.2",
     "nyc": "^13.3.0"
   },
   "engines": {
-    "node": ">=6"
+    "node": ">=8"
```

Comment:
not sure about this one since we have some services using this that may be running on node 6, @dial-once/developers can you check?

Reply:
I do not really see the point of bumping the node version here, since this module was not rewritten to use async/await or any other feature that would not be run on v6. Or am I mistaken? @shamil

Reply:
@Spring3 right, the reason I bumped node version, is that in April 30th version 6 will reach end of life. 

https://github.com/nodejs/Release#release-schedule

Reply:
I think this is not enough reason to discard it.